turbinia/workers/__init__.py

```diff
@@ -599,11 +617,31 @@ def run_wrapper(self, evidence):
     """
     log.debug('Task {0:s} {1:s} awaiting execution'.format(self.name, self.id))
     evidence = evidence_decode(evidence)
+    try:
+      self.result = self.setup(evidence)
+      self.result.update_task_status(self, 'Queued')
+    except Exception as exception:
```

Comment:
I don't think we need to do as broad of Exception and error handling here as we do below because we're not actually executing the Task code here, we're just instantiating the result (which in turns instantiates output_manager and state_managers).  I took a quick look at those methods, and they only raise TurbiniaException, so I think we can just catch that here, and then log errors as appropriate and return `self.result.serialized()` (no need to do all the checks below to make sure we have a valid result object, etc).

Just FYI: Catching the base`Exception` class should actually be quite rare in python.  See "never use catch-all" section here: http://google.github.io/styleguide/pyguide.html#244-decision